.github/workflows/ci.yml

```diff
@@ -207,6 +211,10 @@ jobs:
     needs:
       - fmt
       - check-file-change
+    if: >
+      needs.check-file-change.outputs.parachain_src == 'true' ||
+      needs.check-file-change.outputs.tee_src == 'true' ||
+      needs.check-file-change.outputs.tee_test == 'true'
```

Comment:
This is not needed, this step should always be executed, the building itself however not (check the sub-steps below)

Reply:
Actually I think it is good to have. Because the build/pull step is going to be used in test. But test also depends on these conditions. If the test is skipped, the build/pull here can also be skipped.

Reply:
come to alignment after offline discussion.